### PR TITLE
Wizard recipe: AlgRemez-v0.1.0

### PR DESCRIPTION
**Pull request by cometscome**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: AlgRemez
* Version: v0.1.0

@staticfloat please review and merge.

### CODE REVIEW

File: A/AlgRemez/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+sources = [
+    GitSource("https://github.com/maddyscientist/AlgRemez.git", "bdacfd1b9a7f5c34275da5f3a5a3f7f8cbbf8d9c")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd AlgRemez/src/
+CPLUS_INCLUDE_PATH=/workspace/destdir/include make
+cp test $prefix/algremez
```
Comment (giordano) [inline A/AlgRemez/build_tarballs.jl:18]:
```suggestion
make CPLUS_INCLUDE_PATH=${includedir}
mkdir -p ${bindir}
cp test ${bindir}/algremez${exeext}
```

File: A/AlgRemez/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    GitSource("https://github.com/maddyscientist/AlgRemez.git", "bdacfd1b9a7f5c34275da5f3a5a3f7f8cbbf8d9c")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd AlgRemez/src/
+make CPLUS_INCLUDE_PATH=${includedir}
+mkdir -p ${bindir}
+cp test ${bindir}/algremez${exeext}
```
Comment (giordano) [inline A/AlgRemez/build_tarballs.jl:19]:
```suggestion
mkdir -p ${bindir}
make INCLIST=-I${includedir} LDFLAGS="-L${libdir} -lmpfr -lgmp" BIN=${bindir}/algremez${exeext}
```

**Comment by cometscome** [comment]
@giordano Thank you for revising build_tarballs.jl !

**Comment by giordano** [comment]
Thanks for your contribution!

**Pull request merged**